packages/itwinui-react/src/core/Buttons/IconButton.tsx

```diff
@@ -71,7 +83,12 @@ export const IconButton = React.forwardRef((props, ref) => {
   );
 
   return label ? (
-    <Tooltip content={label} ariaStrategy='none'>
+    <Tooltip
+      content={label}
+      ariaStrategy='none'
+      placement={buttonGroupOrientation === 'vertical' ? 'right' : 'top'}
```

Comment:
In the viewer, I think the vertical button group is placed in the bottom right corner (not sure). So would `left` be better when `buttonGroupOrientation === 'vertical'`?

Reply:
But toolbars can also be on the left side, so there is no right or wrong answer. The important part is not obscuring the adjacent buttons in the button-group. For anything else, the user should configure `labelProps`.